Classes/Model/BasePDF.php

```diff
@@ -96,6 +96,11 @@ class BasePDF extends Fpdi
      */
     protected $currentPageFormat = '';
 
+    /**
+     * @var int
+     */
+    protected $currentTemplateId = 0;
```

Comment:
Isn't this variable already declared in `FpdfTplTrait`?

Reply:
No it is not. PHPStorm would have told me.

Reply:
Ah, I see..a different trait is included, sry!